src/react/GraffyContext.js

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Graffy from '@graffy/core';
```

Comment:
I had not considered this when proposing to use JSDoc. This will result in '@graffy/core' being added as a dependency to this package, which might cause problems for apps using it.